xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4355,6 +4355,21 @@ bool CVideoPlayer::OnAction(const CAction &action)
       case ACTION_NEXT_ITEM:
       case ACTION_CHANNEL_UP:
       {
+        if (m_pInputStream->IsStreamType(DVDSTREAM_TYPE_PVRMANAGER)) 
```

Comment:
In v18 there won't be a streamtype PVR anymore. I would just drop this limitation.